src/main/scala/ai/privado/entrypoint/RuleValidator.scala

```diff
@@ -26,40 +31,59 @@ object RuleValidator extends CommandProcessor {
   }
 
   def validateRules(): Either[String, Unit] = {
-
     getRulesPathFromConfig match {
       case Left(()) =>
         logger.error("Failed to validate rules directory")
         Left("")
-      case Right(yamlDirectory) =>
-        try {
-          val validationFailures = YamlFileValidator.validateDirectory(yamlDirectory)
-          var errorsFound        = 0
-          validationFailures.toList.foreach(vf => {
-            errorsFound += 1
-            println(PRETTY_LINE_SEPARATOR)
-            println(f"File: ${vf.file.pathAsString} has following errors:")
-            vf.validationMessages.foreach(msg => println(msg))
-          })
-          println(PRETTY_LINE_SEPARATOR)
-          errorsFound match {
-            case 0 =>
-              println(s"Completed command: ${CommandConstants.VALIDATE} for directory: $yamlDirectory, with no errors")
-            case 1 =>
-              println(s"Completed command: ${CommandConstants.VALIDATE} for directory: $yamlDirectory, with 1 error")
-            case _ =>
-              println(
-                s"Completed command: ${CommandConstants.VALIDATE} for directory: $yamlDirectory, with ${errorsFound.toString} errors"
-              )
-          }
-          Right(())
-        } catch {
-          case ex: Exception =>
-            println("Failed to validate rules", ex)
-            logger.debug("Failed to validate rules", ex)
-            Left(ex.toString)
-        }
+      case Right(yamlDirectory) => validateDirectory(yamlDirectory)
+    }
+  }
+
+  def validateConfig(): Either[String, Unit] = {
+    getConfigPathFromConfig match {
+      case Left(_) =>
+        logger.error("Failed to validate rules directory")
+        Left("")
```

Comment:
@abhstabs If we don't want to pass any information `Some` and `None` is a better option here